index.html

```diff
@@ -58,5 +58,40 @@ <h1 class="my_name">I’m Karam Glad to see you!</h1>
              <div class="ellipse8"></div>
              <div class="ellipse15"></div>
          </div>
+         <div class="flex-botu">
+            <div class="resume">Get my resume</div>
+         </div>
+         <ul class="border-lang">
+             <li>Langguages</li>
+         </ul>    
+         <ul class="lang-img">
+             <li><a href="#" class="js-icon"><img src="source/js-icon.png" alt="js-icon"><span>JavaSript</span></a></li>
+             <li><a href="#" class="html-icon"><img src="source/html-icon.png" alt="html-icon"><span>HTML</span></a></li>
+             <li><a href="#" class="css-icon"><img src="source/css-icon.png" alt="css-icon"><span>CSS</span></a></li> 
+         </ul>        
+         <ul class="fr-work">
+             <li class="border-frem">Frameworks</li>
+             <li class="border-skill">Skills</li>
+         </ul>
+         <div class="work-sec">
+             <div class="snapshoot-img"></div>
+             <ul class="tonic">
+                 <li>Tonic</li>
+             </ul>
+             <ul class="de-comp">
+                 <li class="conopy">CANOPY</li>
+                 <li class="back-end">Back End Dev</li>
+                 <li class="year-nu">2015</li>
+             </ul>
+             <div class="paragraph"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</div>        
+             <div class="butto">
+                 <button class="htm">html</button>
+                 <button class="css-st">css</button>
+                 <button class="java-scr">javaScript</button>
+             </div>
+             <div class="see-proj">
+                 <button class="see-pr">See Project</button>
+             </div> 
+         </div>
```

Comment:
- [ ] As requested by the former code reviewer, please implement the changes above. 

***See Design***
![satD](https://user-images.githubusercontent.com/65852621/134752020-231a9dce-444d-438a-ac52-606b854824af.png)
- [ ] For each project, align the content to the left as in the design. 
- [ ] Add complete styles to the about me section specifically the categories (languages, skills, frameworks). Make this section math the given design.
- [OPTIONAL ] Please kindly style the buttons for enabled, hover and pressed states.
- [OPTIONAL] Please make sure your social media icons have uniform size.